ddtrace/llmobs/_integrations/langchain.py

```diff
@@ -194,6 +205,40 @@ def _llmobs_set_meta_tags_from_chain(
             except TypeError:
                 log.warning("Failed to serialize chain output data to JSON")
 
+    def _llmobs_set_meta_tags_from_embedding(
+            self,
+            span: Span,
+            input_texts: Union[str, List[str]],
+            output_embedding: Union[List[float], List[List[float]], None],
+            error: bool = False,
+            is_workflow: bool = False,
+    ) -> None:
+        span.set_tag_str(SPAN_KIND, "workflow" if is_workflow else "embedding")
+        span.set_tag_str(MODEL_NAME, span.get_tag(MODEL) or "")
+        span.set_tag_str(MODEL_PROVIDER, span.get_tag(PROVIDER) or "")
+
+        input_tag_key = INPUT_VALUE if is_workflow else INPUT_DOCUMENTS
+        output_tag_key = OUTPUT_VALUE
+
+        if input_texts is not None:
+            try:
+                formatted_inputs = self.format_io(input_texts)
+                if isinstance(input_texts, str):
+                    span.set_tag_str(input_tag_key, formatted_inputs)
+                else:
+                    span.set_tag_str(input_tag_key, json.dumps(self.format_io(input_texts)))
```

Comment:
We need to do slightly different formatting of the input documents/value depending on if the span is a workflow or embedding type span (i.e. embedding spans --> input_documents containing Document() objects, workflow spans --> input_value containing json serialized input texts). I think this is currently formatting all input_value/documents to json text